Tom dynamo implement

tswhitworth opened the pull request.

robert-daniels commented:
@tswhitworth , I'm showing deletions of the random review lines. You did a great job on that functionality! Is it really slated for deletion?

tswhitworth commented:
> @tswhitworth , I'm showing deletions of the random review lines. You did a great job on that functionality! Is it really slated for deletion?

I deleted a function called random review, but I moved it into the home function is that I didn't need to create another route in url.py
